CHANGELOG.md

```diff
@@ -49,6 +49,9 @@
 - Bump Android SDK from v7.18.0 to v7.18.1 ([#4329](https://github.com/getsentry/sentry-react-native/pull/4329))
   - [changelog](https://github.com/getsentry/sentry-java/blob/main/CHANGELOG.md#7181)
   - [diff](https://github.com/getsentry/sentry-java/compare/7.18.0...7.18.1)
+- Bump JavaScript SDK from v8.40.0 to v8.41.0 ([#4351](https://github.com/getsentry/sentry-react-native/pull/4351))
```

Comment:
Let's add small `### Changes` section and add `- Falsy values of `options.environment` (empty string, undefined...) default to `production``

Reply:
Good point 👍 
thank you for catching this and adding a changelog @krystofwoldrich 🙇